src/addons/presenter.js

```diff
@@ -248,33 +251,44 @@ class PresenterMediator extends React.PureComponent {
     return this.model.value
   }
 
+  // Do not allow transfer across repos. Check to for inheritence by comparing
+  // the common history object shared between repos
   hasParent(): boolean {
-    // Do not allow transfer across repos. Check to for inheritence by comparing
-    // the common history object shared between repos
+    // No need to check if there isn't a parent context setup
+    if (!this.context.parent) {
+      return false
+    }
+
     return get(this.repo, 'history') === get(this.context, ['repo', 'history'])
   }
```

Comment:
There might be two separate histories, like if you mount one microcosm app into another (we hit this on iContact with the email editor and photo manager).